src/main/scala/com/atomist/rug/ts/TypeScriptInterfaceGenerator.scala

```diff
@@ -118,24 +118,6 @@ class TypeScriptInterfaceGenerator(
 
   val typeSort: (Typed, Typed) => Boolean = (a, b) => a.name <= b.name
 
-//  DO NOT DELETE
```

Comment:
Well, that's what source control is for! Leaving in old code makes no sense IMO.